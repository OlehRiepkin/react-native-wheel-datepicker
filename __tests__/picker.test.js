import { shallow } from "enzyme";
import React from "react";

import Picker from "../src/picker";

test("basic scenario", () => {
  const onValueChange = jest.fn();
  const wrapper = shallow(
    <Picker
      selectedValue={1}
      pickerData={[1, 2, 3, 4, 5, 6]}
      onValueChange={onValueChange}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
