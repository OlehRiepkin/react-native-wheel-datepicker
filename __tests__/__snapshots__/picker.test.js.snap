// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic scenario 1`] = `
<PickerIOS
  itemSpace={20}
  itemStyle={null}
  onValueChange={[Function]}
  selectedValue={1}
  style={
    Array [
      Object {
        "backgroundColor": "#d3d3d3",
        "height": 220,
      },
      null,
    ]
  }
  textColor="#333"
  textSize={26}
>
  <PickerIOSItem
    key="0"
    label="1"
    value={1}
  />
  <PickerIOSItem
    key="1"
    label="2"
    value={2}
  />
  <PickerIOSItem
    key="2"
    label="3"
    value={3}
  />
  <PickerIOSItem
    key="3"
    label="4"
    value={4}
  />
  <PickerIOSItem
    key="4"
    label="5"
    value={5}
  />
  <PickerIOSItem
    key="5"
    label="6"
    value={6}
  />
</PickerIOS>
`;
